tailscale/wrapper.sh

```diff
@@ -9,6 +9,3 @@ if [ "${ENABLE_TAILSCALE}" = true ]; then
     /tailscale/tailscaled --state=/var/lib/tailscale/tailscaled.state --socket=/var/run/tailscale/tailscaled.sock &
     /tailscale/tailscale up --authkey=${TAILSCALE_AUTHKEY} --hostname="${TAILSCALE_APP_NAME}" --accept-routes=true
 fi
-
-echo "Starting Realtime"
-sudo -E -u nobody /app/bin/server
```

Comment:
Let's rename this file if it is not a wrapper anymore

Reply:
@abc3 it's still a wrapper for tailscale though and it's name-spaced appropriately: `tailscale/wrapper.sh`